Lombiq.ContentEditors.Samples/Readme.md

```diff
@@ -8,8 +8,8 @@ For general details about and usage instructions see the [root Readme](../Readme
 
 This sample project demonstrates:
 
-- How to create an async editor provider for a content type. Once you've enabled the module you can access the demo editor on this URL: `/Admin/ContentItemAsyncEditor/Employee`.
-- How to create an async editor that can be used on the front-end instead of the Admin UI. Once you've enabled the module you can access the demo editor on this URL: `/FrontEndDemoContentItemAsyncEditor`.
+- How to create an async editor provider for a content type. If you are using the OSOCE application you can try it out on the front-end by clicking on the "Content Editor Samples" and "Employee (admin)" menu items. It will redirect you to the Admin UI where you can click through an async editor for the Employee content type that contains two editor pages. You can also access the demo editor on this URL: `/Admin/ContentTypes/Employee/ContentItemAsyncEditor`.
+- How to create an async editor that can be used on the front-end instead of the Admin UI. You can try it out by clicking on the "Content Editor Samples" and "Support Ticket (front-end)" menu items. It will redirect you to an editor where you can click through an async editor for the Support Ticket content type that contains two editor pages. Instead of using the Admin UI, it uses a custom front-end editor that is initialized by a custom controller (`FrontEndDemoContentItemAsyncEditorController`). You can also access the demo editor on this URL: `/FrontEndDemoContentItemAsyncEditor`.
```

Comment:
```suggestion
- How to create an async editor provider for a content type. If you are using the OSOCE application you can try it out on the front-end by clicking on the "Content Editor Samples" and "Employee (admin)" menu items. It will redirect you to the Admin UI where you can click through an async editor for the Employee content type that contains two editor pages. You can also access the demo editor on this URL: _/Admin/ContentTypes/Employee/ContentItemAsyncEditor_.
- How to create an async editor that can be used on the front-end instead of the Admin UI. You can try it out by clicking on the "Content Editor Samples" and "Support Ticket (front-end)" menu items. It will redirect you to an editor where you can click through an async editor for the Support Ticket content type that contains two editor pages. Instead of using the Admin UI, it uses a custom front-end editor that is initialized by a custom controller (`FrontEndDemoContentItemAsyncEditorController`). You can also access the demo editor on this URL: _/FrontEndDemoContentItemAsyncEditor_.
```